ci_config.json

```diff
@@ -23,6 +23,17 @@
     "graphite2",
     "mocklibc"
   ],
+  "apache-orc": {
+    "_comment": "Tests fail on Alpine. Windows symbol visibility broken",
+    "build_on": {
+      "alpine": false,
+      "msys2": false,
+      "windows": false
+    },
```

Comment:
Are these still broken, and if so, is it expected that they'll be fixed?  `build_on` is generally meant for wraps that permanently won't support a platform, in which case the project's `meson.build` is expected to call `error()` on those platforms with an appropriate message.